kalamine/tpl/base.ahk

```diff
@@ -61,10 +61,22 @@ SetTimer, ShowTrayTip, -1000  ; not working
 
 global DeadKey := ""
 
+UnicodeUpperIfCapsLock(unicode:="") {
+  CapsLockOn := GetKeyState("CapsLock", "T") 
+  if %CapsLockOn% {
+    unicode := Chr("0x" SubStr(unicode, 3, 4))
+    StringUpper, unicode, unicode
+  }
+  return unicode
+}
```

Comment:
If we’re factorizing this much, I’d suggest including the `Send` command here as well rather than returning a value, and giving a better name to this function (e.g. `SendChar`).

Nitpick: the empty default value does not make much sense here, does it?